$OpenBSD: patch-src_libgmtk_gmtk_media_tracker_c,v 1.2 2011/12/06 16:05:44 dcoppa Exp $

Fix seeking in the media_tracker when the mouse button is released
(upstream svn revision r66 of gmtk)

Fix problem with seeking in the tracker
(upstream svn revision r48 of gmtk)

--- src/libgmtk/gmtk_media_tracker.c.orig	Tue Jun  7 21:26:37 2011
+++ src/libgmtk/gmtk_media_tracker.c	Tue Dec  6 15:55:41 2011
@@ -168,9 +168,6 @@ static gboolean gmtk_media_tracker_button_release(GtkW
         gtk_range_set_value(GTK_RANGE(GMTK_MEDIA_TRACKER(tracker)->scale), position);
         g_signal_emit_by_name(tracker, "value-changed", (gint) (100 * position));
 
-        difference = (GMTK_MEDIA_TRACKER(tracker)->length * position) - GMTK_MEDIA_TRACKER(tracker)->position;
-        g_signal_emit_by_name(tracker, "difference-changed", difference);
-
         GMTK_MEDIA_TRACKER(tracker)->mouse_down = FALSE;
 
         if (GMTK_MEDIA_TRACKER(tracker)->length > 0.0) {
@@ -196,9 +193,6 @@ static gboolean gmtk_media_tracker_motion_notify(GtkWi
         gtk_range_set_value(GTK_RANGE(GMTK_MEDIA_TRACKER(tracker)->scale), position);
         g_signal_emit_by_name(tracker, "value-changed", (gint) (100 * position));
         difference = (GMTK_MEDIA_TRACKER(tracker)->length * position) - GMTK_MEDIA_TRACKER(tracker)->position;
-        if (ABS(position) > 15)
-            g_signal_emit_by_name(tracker, "difference-changed", difference);
-
     } else {
         if (GMTK_MEDIA_TRACKER(tracker)->length > 0.0) {
             tip = gm_seconds_to_string(GMTK_MEDIA_TRACKER(tracker)->length * ((gdouble) event->x / alloc.width));
